src/home_robot/environment.yml

```diff
@@ -43,5 +51,4 @@ dependencies:
     - imageio
     - pygifsicle
     - pynput
-    - git+https://github.com/openai/CLIP.git
```

Comment:
I believe real world OVMM needs this when completely unseen object names are inputs

Reply:
Added this back.